### ISSUE TITLE
Still get a collapse result from wav2vec2

### ISSUE THREAD

**Issue by andylida** [facebookresearch/fairseq#4530]
### Before asking:
1. search the issues. Already. Tested some solutions from other issues but don't work.

#### What is your question?
I tried some advice from #3718 and #3661 which are mainly asked to decrease the learning rate and set the optimize frequency. I deceased LR to 1e-5 and optimize freq to 32(2GPUs). The result now still looks bad and here is my result. I observed that contrastive loss just fell to decrease and perplexity went randomly as #3735 showed. 

![image](https://user-images.githubusercontent.com/48825609/177105268-b7220397-84f2-4fc3-a657-62d5fe83c91c.png)

![image](https://user-images.githubusercontent.com/48825609/177105296-8bb87d96-9ff5-41a1-ab97-0c90bd02698c.png)

![image](https://user-images.githubusercontent.com/48825609/177105325-b6ebbb44-d1b6-47ee-af80-82289703c5b9.png)

![image](https://user-images.githubusercontent.com/48825609/177105357-69ea9c73-8ef6-485f-9da3-8f6799ad57ec.png)

The red one is valid loss and green one is training loss.

#### What have you tried?
I'm now training VCTK dataset which has only 50h data and maybe it's too small for a self-training task, however, it seems to be something could be fixed by a lower batch token or smaller code num. But I still want to know, what exactly the **A Proper Loss Curve** looks like? I didn't see any properly functioned loss from those issues. it will be appreciated if anyone share some good results. Thanks!

**Comment by andylida** [comment]
I guess it just don't work with this few amount of data.
